packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -88,11 +88,20 @@ export class InsecureUrlResolver implements IUrlResolver {
         }
     }
 
-    private resolveHelper(documentId: string) {
+    private resolveHelper(documentId: string, uri: string) {
         const encodedTenantId = encodeURIComponent(this.tenantId);
         const encodedDocId = encodeURIComponent(documentId);
+        let documentUrl: string;
+        if (uri !== "") {
+            const pathArray = uri.split("/");
+            const encodedPathArray = pathArray.map((pathPart) => encodeURIComponent(pathPart));
+            const encodedUri = encodedPathArray.join("/");
+            documentUrl = `fluid://${new URL(this.ordererUrl).host}/${encodedTenantId}/${encodedDocId}/${encodedUri}`;
+        }
+        else {
+            documentUrl = `fluid://${new URL(this.ordererUrl).host}/${encodedTenantId}/${encodedDocId}`;
```

Comment:
>documentUrl = `fluid://${new URL(this.ordererUrl).host}/${encodedTenantId}/${encodedDocId}`; [](start = 12, length = 92)

This is need for both the if-else branches. So, I would rather do something like to avoid code duplication:
let documentUrl = `fluid://${new URL(this.ordererUrl).host}/${encodedTenantId}/${encodedDocId}`;
if (uri === "") {
  // Get encodedUri
  documentUrl += `${encodedUri}`.
}